_posts/2017-12-04-gis-coordinate-systems.md

```diff
@@ -0,0 +1,106 @@
+excerpt: What are the coordinate systems and how should I take them in account on my GIS development work? 
+tags:
+- GIS
+---
+
+The coordinate systems are probably one of the most confusing areas of the GIS systems. What are the coordinate systems and how should I take them in account on my development work? In this post I will cover some basic principles related to this area.
+
+First, definitions for the terms:
+
+- *__A coordinate system__ is a method for identifying the location of a point on the earth. Most coordinate systems use two numbers (= a coordinate) to identify the location of a point.*
+- *__A map projection__ is a method for taking the curved surface of the earth and displaying it on something flat, like a computer screen or a piece of paper. This mathematical transformation is commonly referred to as a map projection.*
+
+In the other words, coordinate systems are required to give a numerical value for a geographic location on the earth surface. Projections are required to present a round 3D ball as a flat 2D plane. 
+
+We can remember from the school mathematic lessons that presenting the ball on the plate is not so easy issue to solve, and job can be done by multiple different ways. It is not surprising that during the decades, there has been introduced tens of coordinate systems and projections. It seems that every map maker has been different kind of the vision about the our world :) Some popular coordinate systems and projections are for example, WebMercator, WGS84 and ETRS-TM35FIN.
+
+![](/img/gis-coordinate-systems/projections.jpg "Image: Compare Map Projections. Tobias Jung.  https://map-projections.net/imglist.php")
+
+## WebMercator projection
+
+In a modern web mapping solution the most popular projection is so called WebMercator projection. The WebMercator is used by the every major mapping provider such as Google Maps, Bing, Here etc. The WebMercator is based to the Mercator projection, that was presented by Gerardus Mercator in 1569. At 2005 Google introduced its own Mercator variant named by the WebMercator that rapidly become de facto standard of the modern mapping solutions.
+
+I think that the practical reason for popularity of the WebMercator is the fact that it is capable to cover the whole world in a reasonable format, it looks nice for eye and works well on the web and the digital devices. On the other hand, known problem of the WebMercator is its heavy distortion of the north and the south areas of the world. Equator is represented as a natural dimensions but distances start to stretch when traveling to south and north. For example, Greenland seems to be almost same since as Africa. In reality, the Greenland is about 1/14 size.
+
+In the Internet there has been debad related to popularity of the WebMercator projection. Maybe the one reason is geopolitical, since the WebMercator presents industrialized countries such as US and Europe unrealistically large related to Africa and other equator countries.
```

Comment:
Oisko sujuvampi jotenkin näin?

People have argued about the popularity of the WebMercator projection...

Maybe the one kuulostaa omaan korvaan myös oudolta, laittaisin ilman artikkelia Maybe one reason...